sdk/core/core-util/.eslintrc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "plugins": ["@azure/azure-sdk"],
+  "extends": ["plugin:@azure/azure-sdk/azure-sdk-base"],
+  "rules": {
+    "@azure/azure-sdk/ts-package-json-module": "off"
```

Comment:
since we're not building tests anymore, this changed our dist-esm paths and the rule here doesn't account for it.